gulpfile.js

```diff
@@ -160,6 +161,7 @@ createTask('make-extension', makeExtension);
 createTask('nailgun-start', nailgunStart);
 createTask('nailgun-stop', nailgunStop);
 createTask('performance', performance);
+createTask('performance-urls', performanceUrls);
```

Comment:
Created the task, but not sure how/where I should incorporate this into our CI.

Thoughts @estherkim ?

Reply:
You can call it here - 
https://github.com/ampproject/amphtml/blob/master/build-system/pr-check/checks.js

Reply:
Does adding it here seem appropriate? https://github.com/ampproject/amphtml/blob/master/build-system/pr-check/checks.js#L71

Reply:
Yep, doesn't make sense to create a BUILD_TARGET for this. Make sure to add it to master push builds too, in here https://github.com/ampproject/amphtml/blob/master/build-system/pr-check/checks.js#L47

Reply:
Cool, is there a way to test this?

Reply:
Travis logs show that this succeeded: https://travis-ci.org/github/ampproject/amphtml/jobs/681697643#L428